web/src/layout/leftpanel/countrypanel/index.jsx

```diff
@@ -57,20 +58,42 @@ const CountryPanelStyled = styled.div`
     margin-bottom: 60px; /* height of time-slider plus padding*/
   }
 `;
+//The changes are all WIP, will find the optimal solution later
+// const getExchangesForZone = (zoneId, exchanges) => {
+//   const exchangesKeysForZone = Object.keys(exchanges).filter(
+//     (exchange) => exchange.split('->')[0] === zoneId || exchange.split('->')[1] === zoneId
+//   );
+//   const exchangesForZone = exchangesKeysForZone.map((exchangeKey) => exchanges[exchangeKey]);
+
+//   return exchangesForZone;
+// };
 
 const CountryPanel = ({ electricityMixMode, isMobile, tableDisplayEmissions, zones }) => {
   const { __ } = useTranslation();
   const isLoadingHistories = useSelector((state) => state.data.isLoadingHistories);
   const isLoadingGrid = useSelector((state) => state.data.isLoadingGrid);
   const failedRequestType = useSelector((state) => state.data.failedRequestType);
-
+  const timeAggregate = useSelector((state) => state.application.selectedTimeAggregate);
   const trackEvent = useTrackEvent();
   const history = useHistory();
   const location = useLocation();
   const { zoneId } = useParams();
-  const timeAggregate = useSelector((state) => state.application.selectedTimeAggregate);
-
-  const data = useCurrentZoneData() || {};
+  //const selectedZoneTimeIndex = useSelector((state) => state.application.selectedZoneTimeIndex);
+  const showAggregated = useAggregatesEnabled();
+  const zoneViewLeftPanelData = useCurrentZoneData() || {};
+  //const allExchanges = useSelector((state) => state.data.exchanges);
+  //const exchangesForZone = getExchangesForZone(zoneId, allExchanges);
+
+  // const countryViewLeftPanelData = {
+  //   ...zoneViewLeftPanelData,
+  //   exchange: zoneForTimePeriod?.exchange,
+  //   exchangeCapacities: zoneForTimePeriod?.exchangeCapacities,
+  //   exchangeCo2Intensities: zoneForTimePeriod?.exchangeCo2Intensities,
+  // };
+
+  const leftPanelData = showAggregated ? zoneViewLeftPanelData : zoneViewLeftPanelData;
```

Comment:
I assume this will show countryViewLeftPanelData when `true` when the PR is ready?

Reply:
Yeah, everything in this file is just testing to see what data was available. Markus is coming through with a sql query so we can change the back-end api to get everything we need to show exchange info on the country panel.